qiskit/backends/local/qasm_simulator_py.py

```diff
@@ -271,7 +271,9 @@ def run(self, qobj):
         Returns:
             LocalJob: derived from BaseJob
         """
-        return LocalJob(self._run_job, qobj)
+        local_job = LocalJob(self._run_job, qobj)
+        local_job.submit()
+        return local_job
 
     def _run_job(self, qobj):
```

Comment:
I wonder if we should simplify all the non-async calls  in the local backends to just `_run(self, qobj)`?

Reply:
Yep, it's kind of repetitive.. I can change this.